File: RemoveDuplicatesfromSortedArray/reDuFSA.py
class Solution:
    def removeDuplicates(self, nums: 'List[int]') -> 'int':
        if len(nums)<=1:
            return len(nums)
        temp = 1;
        for i in range(1,len(nums)):
            if nums[i]!=nums[i-1]:
                nums[temp] = nums[i]
                temp+=1
        return temp
        

File: RemoveDuplicatesfromSortedArray/reDuFSA.php
class Solution {

/**
 * @param Integer[] $nums
 * @return Integer
 */
function removeDuplicates(&$nums) {
    $numNew = [];
    $numNew[] = $nums[0];
    for($i=1;$i<count($nums);$i++){
        if($nums[$i]!=$nums[$i-1]){
            $numNew[] = $nums[$i];
        }
    }
    $nums = $numNew;
    return count($numNew);
}
}